isaric/parsers/stopcovid-russia.toml

```diff
@@ -0,0 +1,601 @@
+
+[[observation]]
+  name = "inability_to_walk"
+  date.ref = "admissionDate"
+  phase = "admission"
+  is_present = { field = "inablewalk_ceoccur_v2", ref = "Y/N" }
+
+[[observation]]
+  name = "loss_of_smell"
+  date.ref = "followupDate"
+  phase = "followup"
+  is_present = { field = "flw_loss_smell", ref = "boolean" }
+
+[[observation]]
+  name = "loss_of_taste"
+  date.ref = "followupDate"
+  phase = "followup"
+  is_present = { field = "flw_loss_taste", ref = "boolean" }
+
+[[observation]]
+  name = "loss_of_smell_or_taste"
+  date.ref = "followupDate"
+  phase = "followup"
+  is_present.combinedType = "any"
+  is_present.fields = [
+    { field = "flw_loss_smell", ref = "boolean" },
+    { field = "flw_loss_taste", ref = "boolean" }
+  ]
+
+[[observation]]
+  name = "chest_pain"
+  date.ref = "followupDate"
+  phase = "followup"
+  is_present = { field = "flw_chest_pains", ref = "boolean" }
+
+[[observation]]
+  name = "abdominal_pain"
+  date.ref = "followupDate"
+  phase = "followup"
+  is_present = { field = "flw_stomach_pain", ref = "boolean" }
+
+[[observation]]
+  name = "diarrhoea"
+  date.ref = "followupDate"
+  phase = "followup"
+  is_present = { field = "flw_diarrhoea", ref = "boolean" }
+
+[[observation]]
+  name = "muscle_aches"
+  date.ref = "followupDate"
+  phase = "followup"
+  # TODO: no flw_mus_pain present, check if phase is correct
+  is_present = { field = "mus_pain", ref = "boolean" }
+
+[[observation]]
+  name = "confusion"
+  date.ref = "followupDate"
+  phase = "followup"
+  is_present = { field = "flw_confusion", ref = "boolean" }
+
+[[observation]]
+  name = "skin_rash"
+  date.ref = "followupDate"
+  phase = "followup"
+  is_present = { field = "flw_skin_rash", ref = "boolean" }
+
+[[observation]]
+  name = "bleeding"
+  date.ref = "followupDate"
+  phase = "followup"
+  is_present = { field = "flw_bleeding", ref = "boolean" }
+
+[[observation]]
+  name = "temperature_celsius"
+  phase = "admission"
+  date.ref = "admissionDate"
+  value.field = "temp_vsorres"
+  value.unit = "celsius"
+  value.source_unit = { field = "temp_vsorresu", values = { 1 = "celsius", 2 = "fahrenheit" } }
+
+[[observation]]
+  name = "heart_rate_bpm"
+  phase = "admission"
+  date.ref = "admissionDate"
+  value.field = "hr_vsorres"
+
+[[observation]]
+  name = "respiratory_rate"
+  phase = "admission"
+  date.ref = "admissionDate"
+  value.field = "rr_vsorres"
+
+[[observation]]
+  name = "oxygen_saturation_percent"
+  phase = "admission"
+  date.ref = "admissionDate"
+  value.field = "oxy_vsorres"
+  context.combinedType = "set"
+  context.excludeWhen = "none"
+  context.fields = [
+    { field = "oxy_vsorresu", values = { 1 = "Room air", 2 = "Oxygen therapy" } },
+  ]
+
+[[observation]]
+  name = "sternal_capillary_refill_time_greater_2s"
+  phase = "admission"
+  date.ref = "admissionDate"
+  is_present = { field = "stercap_vsorres", ref = "Y/N" }
+
+[[observation]]
+  name = "inability_to_walk_scale"
```

Comment:
Just thinking, are the two walking attributes - inability_to_walk and inability_to_walk_scale - related? 

If so, should the inability_to_walk_scale be conditional on the attribute 'inability_to_walk' .. that is, 'inability_to_walk_scale' is valid if the inability_to_walk == yes or true?

Reply:
They are related, but usually they are from the same underlying variable. With inability_to_walk == false, inability_to_walk_scale would still have an entry (corresponding to no inability to walk)